python/mxnet/numpy/multiarray.py

```diff
@@ -1759,22 +1880,45 @@ def swapaxes(a, axis1, axis2):
 
 @set_module('mxnet.numpy')
 def expand_dims(a, axis):
```

Comment:
I don't think this function will have the correct signature after rendering. Please build and check, and override the function signature if needed.

Reply:
Thx and this signature works.